roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py

```diff
@@ -50,9 +50,9 @@
 # Examples don't contain auth parameter for simplicity,
 # look at ovirt_auth module to see how to reuse authentication:
 
-# Gather facts about all scheduling policies with name InClusterUpgrade:
+# Gather facts about all scheduling policies with name ClusterInMaintenance:
 - ovirt_scheduling_policies_facts:
-    name: InClusterUpgrade
+    name: ClusterInMaintenance
```

Comment:
So do we need to add logic here to use InClusterUpgrade for oVirt < 4.1.4 and ClusterInMaintenance for oVirt >= 4.1.4?